submitted_models/ssci_x4_sensor_config_1/model.sdf

```diff
@@ -335,13 +335,11 @@
 					We believe this is based on a Velodyne Puck Lite 
 					which weighs 590 g -->
                 <pose>0 0 -0.05 0 0 0</pose>
-                <update_rate>15</update_rate>
+                <update_rate>10</update_rate>
                 <lidar>
                     <scan>
                         <horizontal>
-                            <!-- Resolution can't be above 16k, see
-                            https://bitbucket.org/ignitionrobotics/ign-sensors/issues/8 -->
-                            <samples>10000</samples>
+                            <samples>1800</samples>
                             <resolution>0.1</resolution>
```

Comment:
I suggest increasing horizontal resolution to 1 as well. Otherwise I guess accuracy could become far inferior to physical sensor.

Reply:
> I suggest increasing horizontal resolution to 1 as well. Otherwise I guess accuracy could become far inferior to physical sensor.

I agree that the parameter should be 1.0 however whenever I experimentally tested the impact of changing `resolution` I could not find any difference between.  This could be indicative of a larger problem with horizontal LIDAR ray resolution.

Reply:
Probably you are right. May be gpu ray sensor doesn't even use that parameter. (Just a thought need to check code to confirm) Or the way we think what that parameter is for, is incorrect.

Reply:
It does not currently factor in that parameter.  I'm looking into the issue a bit more before submitting an issue on the Ignition repositories.